pkgs/tools/graphics/realesrgan-ncnn-vulkan/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub,
+  cmake,
+  vulkan-loader, vulkan-headers, glslang, libgcc, libwebp, ncnn }:
+
+stdenv.mkDerivation rec {
+  pname = "Real-ESRGAN-ncnn-vulkan";
+  version = "0.1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "xinntao";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-eLAIlOl1sUxijeVPFG+NscZGxDdtrQqVkMuxhegESHk=";
+  } + /src;
+
```

```suggestion

  patches = [ ./cmakelists.patch ];
```

This patch seems to at least start the build process:

```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index a234caa..2517b8e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -114,7 +114,10 @@ if(USE_SYSTEM_NCNN)
             include("${GLSLANG_TARGET_DIR}/HLSLTargets.cmake")
         endif()
         include("${GLSLANG_TARGET_DIR}/glslangTargets.cmake")
+	include("${GLSLANG_TARGET_DIR}/SPIRV-Tools/SPIRV-ToolsTarget.cmake")
+	include("${GLSLANG_TARGET_DIR}/SPIRV-Tools-opt/SPIRV-Tools-optTargets.cmake")
         include("${GLSLANG_TARGET_DIR}/SPIRVTargets.cmake")
+	include("${NCNN_TARGET_DIR}/ncnn/ncnn.cmake")
 
         if (NOT TARGET glslang OR NOT TARGET SPIRV)
             message(WARNING "glslang or SPIRV target not found! USE_SYSTEM_NCNN will be turned off.")
```

---

There was more to patch, but that was a nice start, thx